XF.Material/Platforms/Ios/Renderers/MaterialButtonRenderer.cs

```diff
@@ -305,8 +306,34 @@ private void SetupIcon()
 
                 try
                 {
-                    image = UIImage.FromFile(_materialButton.Image.File) ?? UIImage.FromBundle(_materialButton.Image.File);
-
+                    if (_materialButton.Image != null)
+                    {
+                        image = UIImage.FromFile(_materialButton.Image.File) ?? UIImage.FromBundle(_materialButton.Image.File);
+                    }
+                    else if(!(_materialButton.ImageSource?.IsEmpty ?? true))
```

Comment:
Why does this need to be done? Also why only on iOS?

Reply:
Well, the android does not have such loading of Image, ImageSource. Only iOS for some reason ? I wanted to ask you for this.
Well, I guessed you want backwards compatibility or not ?

Reply:
Hmm yeah i'm not super sure about a nice solution. In MediaManager i had the same problem and implemented it in all platforms. See:

Cross: https://github.com/Baseflow/XamarinMediaManager/blob/develop/MediaManager.Forms/ImageSourceExtensions.cs

Android: https://github.com/Baseflow/XamarinMediaManager/blob/develop/MediaManager.Forms/Platforms/Android/ImageSourceExtensions.cs

All the other platforms have an implementation as well.

Reply:
Oh, I was rather talking about SetupIcon method and that your code only loads image in iOS, but not in android. 

But yeah, that's my bad, I was short-sighted and did not think it about further usages of this ImageSource to UIImage conversion. Should have created a new class for extensions. Sure, I will move it to extension class of ImageSource.